src/organisms/Animal.js

```diff
@@ -0,0 +1,76 @@
+import { Organism } from '../Organism';
+import { getRandom } from '../utilities/getRandom';
+
+export class Animal extends Organism {
+  constructor(strength, initiative, positionY, positionX, age = 0) {
+    super(strength, initiative, positionY, positionX, age);
+  }
+
+  async action(board) {
+    const directions = [
+      { positionY: -1, positionX: -1 },
+      { positionY: 0, positionX: -1 },
+      { positionY: 1, positionX: -1 },
+      { positionY: -1, positionX: 0 },
+      { positionY: 1, positionX: 0 },
+      { positionY: -1, positionX: 1 },
+      { positionY: 0, positionX: 1 },
+      { positionY: 1, positionX: 1 },
+    ];
+
+    const availableDirections = directions.filter((direction) => {
+      const newY = this.positionY + direction.positionY;
+      const newX = this.positionX + direction.positionX;
+      return board.isValidPosition(newY, newX);
+    });
```

Comment:
Move this to the `this.getAvailableDirections` method. Override it for the Antelope.